exporter/otlpexporter/config.go

```diff
@@ -29,3 +29,10 @@ type Config struct {
 
 	configgrpc.GRPCClientSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
 }
+
+var _ config.Exporter = (*Config)(nil)
+
+// Validate checks if the exporter configuration is valid
+func (cfg *Config) Validate() error {
+	return nil
```

Comment:
Check for endpoint here 
https://github.com/open-telemetry/opentelemetry-collector/blob/c5ddd0942f736e37eda789211090de1eb02cde3d/exporter/otlpexporter/otlp.go#L50-L52